apps/insights/src/components/Overview/index.tsx

```diff
@@ -42,41 +42,23 @@ export const Overview = async () => {
 
   return (
     <div className={styles.overview}>
-      <h1 className={styles.header}>Overview</h1>
       <Cards>
         <ChartCard
-          header="Total Volume Traded"
+          header="Price Feeds"
           variant="primary"
-          data={totalVolumeTraded.map(({ date, volume }) => ({
-            x: date,
-            displayX: <FormattedDate value={date} />,
-            y: volume,
-            displayY: (
-              <FormattedNumber
-                value={volume}
-                currency="usd"
-                style="currency"
-                notation="compact"
-              />
-            ),
-          }))}
+          href="/price-feeds"
+          chartClassName={styles.priceFeedsChart}
+          data={feedCounts}
           miniStat={
             <ChangePercent
-              previousValue={totalVolumeTraded.at(-2)?.volume ?? 0}
-              currentValue={totalVolumeTraded.at(-1)?.volume ?? 0}
-            />
-          }
-          stat={
-            <FormattedNumber
-              value={totalVolumeTraded.at(-1)?.volume ?? 0}
-              currency="usd"
-              style="currency"
-              notation="compact"
+              previousValue={feedCounts.at(-2)?.y ?? 0}
+              currentValue={feedCounts.at(-1)?.y ?? 0}
             />
           }
+          stat={feedCounts.at(-1)?.y}
         />
         <ChartCard
-          header="Publishers Onboarded"
+          header="Publishers"
```

Comment:
This should remain "Publishers Onboarded" -- we have 2 different numbers for publishers that we highlight in different places:

- "Publishers Onboarded" refers to the number of publishers who _can_ publish / be permissioned
- "Active Publishers" refers to the number of publishers who are _actively_ permissioned in pythnet and are pushing data

We used to just have the word "Publishers" here but it was confusing because people would see different counts in different places.

Obviously I don't care about the exact terminology, but the distinction needs to exist.